src/Prana/Types.hs

```diff
@@ -27,7 +27,7 @@ data Exp
   --
   = AppE Exp Exp -- ^ Apply a function to an argument.
   | LamE LocalVarId Exp -- ^ A lambda.
-  | CaseE Exp VarId Typ [Alt] -- ^ A case analysis.
+  | CaseE Exp LocalVarId Typ [Alt] -- ^ A case analysis.
```

Comment:
I also thought about making this change, as it doesn't make sense that `CaseE` would have a global var id. :+1: